frontend/src/components/HospitalInventoryEdit.tsx

```diff
@@ -0,0 +1,194 @@
+      <Form>
+        <h5>Address</h5>
+        <Form.Row>
+          <Form.Group as={Col}>
+            <Form.Control 
+              type="text" 
+              placeholder="street" 
+              name="street"
+              value={hospitalAddress[0] ? hospitalAddress[0].street : ''}
+              onChange={handleAddressChange}
+            />
+          </Form.Group>
+        </Form.Row>
+        <Form.Row>
+          <Form.Group as={Col}>
+            <Form.Control 
+              type="text" 
+              placeholder="city" 
+              name="city"
+              value={hospitalAddress[0] ? hospitalAddress[0].city : ''}
+              onChange={handleAddressChange}
+            />
+          </Form.Group>
+          <Form.Group as={Col}>
+            <Form.Control 
+              type="number" 
+              placeholder="zip" 
+              name="zip"
+              value={hospitalAddress[0] ? hospitalAddress[0].zip : ''}
+              onChange={handleAddressChange}
+            />
+          </Form.Group>
+        </Form.Row>
+        <Form.Row>
+          <Form.Group as={Col} xs={8} md={6}>
+            <Form.Control as="select" 
+              name="country"
+              value={hospitalAddress[0] ? hospitalAddress[0].country : ''}
+              onChange={handleAddressChange}
+            >
+              <option value="India">India</option>
+            </Form.Control>
+          </Form.Group>
+        </Form.Row>
+        <Form.Row>
+          <Form.Group as={Col} xs={8} md={6}>
+            <Button 
+              variant="outline-primary" 
+              size="lg" 
+              block
+              onClick={updateHospitalAddress}
+            >
+              Update Info
+            </Button>{' '}
+          </Form.Group>
+        </Form.Row>
+      </Form>
+
+      <Form>
+        <Form.Row>
+          <Form.Label>Filter By:</Form.Label>
+          <Form.Group as={Col} controlId="formGridState">
+            <Form.Control as="select" value="Choose...">
+              <option>State</option>
+              <option>...</option>
+            </Form.Control>
+          </Form.Group>
+
+          <Form.Group as={Col} controlId="formGridState">
+            <Form.Control as="select" value="Choose...">
+              <option>District</option>
+              <option>...</option>
+            </Form.Control>
+          </Form.Group>
+
+          <Form.Group as={Col} controlId="formGridState">
+            <Form.Control as="select" value="Choose...">
+              <option>India</option>
+            </Form.Control>
+          </Form.Group>
+        </Form.Row>
+      </Form>
+
+      <Form>
+        <h5>Fortis Hospital</h5>
+        <Form.Text className="text-muted">
+          1 June 2020, 05:14:33am
+        </Form.Text>
+        <Form.Row>
+          {inventoryList.map((item, index) => (
+            <Card 
+              className={`text-center ${style.inputCard}`}
+              key={item.resourceType} >
```

Comment:
Excellent. Not index, but the resource type. :) Not that it makes any difference in this case, but it's a very good way of thinking.